docs/framework/windows-workflow-foundation/flowchart-workflows.md

```diff
@@ -40,7 +40,7 @@ To model a basic sequence in which two activities execute in turn, the `FlowStep
     </Assign>
     <FlowStep.Next>
       <FlowStep>
-        <WriteLine Text="["Hello, " & result]"/>
+        <WriteLine Text="Hello, & [result]"/>
```

Comment:
Or should it be:

```suggestion
        <WriteLine Text="Hello, " & [result]/>
```

Reply:
Oh I had severe doubts about this one :|..I will correct it right away. Thank you @gewarren